src/main/java/com/sprint/monew/domain/activity/UserActivityController.java

```diff
@@ -0,0 +1,28 @@
+package com.sprint.monew.domain.activity;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@Slf4j
+@RestController
+@RequestMapping("/api/user-activity")
+@RequiredArgsConstructor
+public class UserActivityController {
+    private final UserActivityService userActivityService;
+
+    @GetMapping("/{userId}")
+    public ResponseEntity<UserActivityDto> getUserActivity(
+            @PathVariable UUID userId,
+            @RequestHeader("Monew-Request-User-ID") UUID headerUserId
```

Comment:
프로토타입 스웨거에서는 헤더로 UserId를 안받고 PathVariable로 받는 것 같은데, 따로 추가하신 부분일까요? 저는 다른 api와 통일성을 위해서 헤더로 받는 것도 좋은 것 같은데, 제공되는 프론트엔드 코드에서 헤더를 보낼까 싶어서요... 그래서 우선 required =  false 로 해두는 건 어떤지 제안드리고 싶습니다!

```suggestion
            @PathVariable UUID userId,
            @RequestHeader(name = "Monew-Request-User-ID", required = false ) UUID headerUserId
```

- 프로토타입 사용자 활동 내역 관리 API 명세: 헤더 필요 x
![image](https://github.com/user-attachments/assets/de610d43-8ebb-4929-a246-9aeb6b0942fe)
- 프로토타입 알림 관리 API 명세: 헤더 필요 o
![image](https://github.com/user-attachments/assets/d0beee65-1c0a-428a-ab9b-8f073fa5bb6b)

Reply:
네 우선 다른 api랑 통일성 때문에 추가해뒀어요. 말씀하신대로 required=false로 해두겠습니다!